subworkflows/nf-core/variantcalling/haplotypecaller/main.nf

```diff
@@ -87,19 +87,14 @@ workflow RUN_HAPLOTYPECALLER {
         // filtered_vcf = JOINT_GERMLINE.out.vcf
         // ch_versions = ch_versions.mix(GATK_JOINT_GERMLINE_VARIANT_CALLING.out.versions)
     } else {
-
-        //Scatter/gather on WGS, on targeted data run all intervals at once to avoid  "A USER ERROR has occurred: Bad input: VCF contains no variants or no variants with INFO score key "CNN_1D"" which happens on small-ish regions frequently
-        if(params.wes){
-            single_sample_in = Channel.empty().mix(haplotypecaller_vcf.join(haplotypecaller_tbi).combine(intervals_bed_combined).map{
-                meta, vcf, tbi, intervals ->
-                [[id:meta.id, patient:meta.patient, sample:meta.sample, gender:meta.gender, status:meta.status, num_intervals:1 ],
-                vcf, tbi, intervals]
-            })
-        }else{
-            single_sample_in = Channel.empty().mix(HAPLOTYPECALLER.out.vcf.join(HAPLOTYPECALLER.out.tbi).join(cram).map{ meta, vcf, tbi, cram, crai, intervals ->
-                [meta, vcf, tbi, intervals]
-            })
-        }
+        single_sample_in = Channel.empty().mix(haplotypecaller_vcf.join(haplotypecaller_tbi).combine(intervals_bed_combined).map{
+            meta, vcf, tbi, intervals ->
+            [[id:meta.id, patient:meta.patient, sample:meta.sample, gender:meta.gender, status:meta.status, num_intervals:1 ],
+            vcf, tbi, intervals]
+        })
+        single_sample_in = Channel.empty().mix(HAPLOTYPECALLER.out.vcf.join(HAPLOTYPECALLER.out.tbi).join(cram).map{ meta, vcf, tbi, cram, crai, intervals ->
+            [meta, vcf, tbi, intervals]
+        })
```

Comment:
```suggestion
```

Reply:
Here you are overwritting single_sample in with one interval file with the sharded intervals causing the mepty vcf, try again with removing this line and then it should hopefully be fine